selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java

```diff
@@ -79,13 +79,16 @@
 
   @BeforeClass
   public void setUp() throws Exception {
-    ide.open(workspace);
+    ide.open(testWorkspace);
   }
 
   @AfterClass
   public void tearDown() throws Exception {
     if (factoryWsName != null) {
       workspaceServiceClient.delete(factoryWsName, user.getName());
+    } else {
+      // workaround to remove test workspace for sure
```

Comment:
When this can happen?

Reply:
factoryWsName can be null when there is a problem with opening workspace created from factory.

Reply:
same https://github.com/eclipse/che/pull/6354#discussion_r140168481